src/main/fc/init.c

```diff
@@ -1009,6 +1010,10 @@ void init(void)
     altHoldInit();
 #endif
 
+#ifdef USE_POS_HOLD_MODE
+    posHoldInit();
```

Comment:
I'm curious about why we should pass `posHoldConfig()`, only because:
- we don't do this in any similar init functions in init.c,
- isn't the posHoldConfig accessible via the inclusion of pg/pos_hold.h?
- it seems to work properly at the moment
?

Reply:
@hydra preffered this style (and IIRC it started before PGs), so it was used quite consistently in Cleanflight. It may be slightly cleaner, but it often leads to redundant local  configuration pointer